src/Client/log.txt

```diff
@@ -1,10 +1,7 @@
-2020-05-04 10:26:32,335 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Successful connection!
-2020-05-04 10:26:32,403 - Imigyen sz�la Domi.Client: - INFO - I got my id: 1
-2020-05-04 10:26:32,404 - Imigyen sz�la Domi.Screen: - INFO - Game started, terrain loaded
-2020-05-04 10:26:42,580 - Imigyen sz�la Domi.Screen: - INFO - ID: 0 Death of player acknowledged.
-2020-05-04 10:26:42,580 - Imigyen sz�la Domi.Screen: - INFO - ID: 0 Player removed from player manager.
-2020-05-04 10:27:07,269 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Client socket closed.
-,571 - Imigyen sz�la Domi.Screen: - INFO - It is our player that died!
-2020-05-04 10:27:07,310 - Imigyen sz�la Domi.Screen: - INFO - ID: 1 Death of player acknowledged.
-2020-05-04 10:27:07,312 - Imigyen sz�la Domi.Screen: - INFO - ID: 1 Player removed from player manager.
-2020-05-04 10:27:09,141 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Client socket closed.
+2020-05-05 11:07:39,165 - Imigyen sz�la Domi.ClientCommunicator: - INFO - Successful connection!
+2020-05-05 11:07:39,168 - Imigyen sz�la Domi.Client: - INFO - I got my id: 1
+2020-05-05 11:07:39,194 - Imigyen sz�la Domi.Screen: - INFO - Game started, terrain loaded
+2020-05-05 11:08:00,728 - Imigyen sz�la Domi.Screen: - INFO - ID: 2 Death of p2020-05-05 11:08:00,734 - Imigyen sz�la Domi.Screen: - INFO - ID: 2 Death of player acknowledged.
+2020-05-05 11:08:00,734 - I2020-05-05 11:08:36,261 - Imigyen sz�la Domi.Screen: - INFO - ID: 3 Death of player acknowledged.
+2020-05-05 11:08:36,261 - Imigyen sz�la Domi.Screen: - INFO - ID: 3 Player removed from player manager.
+migyen sz�la Domi.Screen: - INFO - ID: 3 Player removed from player manager.
```

Comment:
.gitignore not working?

Reply:
Not really... as far as I know if a file is "tracked" by git, it won't release it :( If we delete it and commit the deletion maybe after that gitignore can work.
I try to discard it usually, but it seems not always...